src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
any reason we are removing `static` here? User wouldn't be able to call `Folder::first()`